db/migrate/20151023131950_link_arf_report_directly_to_host.rb

```diff
@@ -0,0 +1,17 @@
+class LinkArfReportDirectlyToHost < ActiveRecord::Migration
+  def up
+    ForemanOpenscap::ArfReport.all.each do |report|
```

Comment:
I have submitted the improved version with batches.
The logic may seem strange, but there was a change from 
```ruby
class ArfReport
  belongs_to :asset, :foreign_key => 'host_id'
end

class Asset
  has_many :arf_reports
  belongs_to :assetable, :polymorphic => true
end

class Host::Managed
  has_one :asset, :as => :assetable
end
```
to
```ruby
class ArfReport
  belongs_to :host
end

class Asset
 belongs_to :assetable, :polymorphic => true
end

class Host::Managed
  has_many :arf_reports
  has_one :asset, :as => :assetable
end
``` 

so the real purpose is to replace asset ids in the 'host_id' column with the ids of the hosts.